cli-feed-v3-3.json

```diff
@@ -0,0 +1,269 @@
+            "localEntryPoint": "func.exe",
+            "itemTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ItemTemplates/2.1.1642",
+            "projectTemplates": "https://www.nuget.org/api/v2/package/Microsoft.Azure.WebJobs.ProjectTemplates/2.1.1642",
+            "projectTemplateId": {
+              "csharp": "Microsoft.AzureFunctions.ProjectTemplate.CSharp.2.x"
+            },
+            "localContainerBaseImage": "DOCKER|mcr.microsoft.com/azure-functions/dotnet:2.0-appservice",
+            "serviceAppSettings": {
+              "FUNCTIONS_EXTENSION_VERSION": "~2",
+              "FUNCTIONS_WORKER_RUNTIME": "dotnet"
+            },
+            "windowsSiteConfig": {
+              "netFrameworkVersion": "v4.0"
+            },
+            "linuxSiteConfig": {
+              "linuxFxVersion": "DOTNET|2.2"
+            }
+          }
+        }
+      },
+      "coreTools": [
+        {
+          "OS": "Linux",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.linux-x64.2.7.3188.zip",
+          "sha2": "f8cff611d0cc2198589a7ff0b1b268549bf86c11afd799098df59f7721d6b771",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "MacOS",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.osx-x64.2.7.3188.zip",
+          "sha2": "baa850c449e9be5fcef8e2bf91883885ec9ab9471437c998701435f5164d72e5",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x64",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.min.win-x64.2.7.3188.zip",
+          "sha2": "1c5c49bfb683e45fc4b44e52e25f63324f9d8e931d47259c55fc66c9b7019fc9",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.win-x86.2.7.3188.zip",
+          "sha2": "4a0342497a606e763aa148a10852d6da91d2d5cc7fdb7e705a0c17df4e038023",
+          "size": "full",
+          "default": "false"
+        },
+        {
+          "OS": "Windows",
+          "Architecture": "x86",
+          "downloadLink": "https://functionscdn.azureedge.net/public/2.7.3188/Azure.Functions.Cli.min.win-x86.2.7.3188.zip",
+          "sha2": "f04accdc23432695786f434835b12558e603866a5c3b6d5c398965d3dccc78e9",
+          "size": "minified",
+          "default": "true"
+        }
+      ]
+    },
+    "3.20.0": {
+      "templates": "https://functionscdn.azureedge.net/public/TemplatesApi/3.1.1648.zip",
+      "workerRuntimeInformation": {
+        "dotnet": {
+          "net5-isolated": {
+            "displayInfo": {
+              "displayName": ".NET 5 (Isolated)",
```

Comment:
[Stacks API](https://github.com/Azure/azure-functions-ux/blob/dev/server/src/stacks/2020-10-01/stacks/function-app-stacks/Dotnet.ts#L9) uses `.NET 5 (non-LTS)`. Would be nice to pick one and be consistent

Reply:
The display name can only have non-translatable parts.  Anything that needs to be displayed with translation will need some boolean/enum properties that tell the client how to assemble the name in a translated form.  ".NET 5" is branding, no translation.  "Isolated" and "non-LTS" both require translation.

Reply:
What if we have a separate "description" property for "Isolated"/"non-LTS". It doesn't solve localizability, but I imagine it would give us more flexibility in the future. Also, VS Code has an established pattern for the UI of "secondary" information, meaning we would do something like this:
<img width="231" alt="Screen Shot 2021-02-24 at 12 01 40 PM" src="https://user-images.githubusercontent.com/11282622/109058679-24d55780-7698-11eb-9ed0-a364a1608c77.png">

Reply:
Ok, I will add a `description` property unless opposed.